### PR TITLE
[ONEM-19552][ONEM-19555] lisa install/uninstall database support added

### PR DESCRIPTION
**Pull request by szymon-polke-red**

### CODE REVIEW

File: LISA/SqlDataStorage.cpp [hunk 1]
```diff
@@ -185,6 +264,99 @@ namespace LISA {
...
+                                       int columns,
+                                       char** columnsTxt,
+                                       char** columnName) -> int {
+
+                                           ASSERT(columnsTxt && columnsTxt[0]);
+                                           try {
+                                               *(static_cast<int*>(appIdx)) = std::stoi(columnsTxt[0]);
+                                           }
+                                           catch(...) {
+                                               // skip silently
```
Comment (piotr-marcinkowski-red) [inline LISA/SqlDataStorage.cpp:303]: why silently? maybe we should at least consider logging it?

File: LISA/SqlDataStorage.cpp [hunk 2]
```diff
@@ -185,6 +264,99 @@ namespace LISA {
...
     }
 
+    void SqlDataStorage::InsertIntoApps(const std::string& type,
+                                        const std::string& id,
+                                        const std::string& appPath,
+                                        const std::string& timeCreated)
+    {
+        std::stringstream query;
+        query << "INSERT INTO apps VALUES(NULL, ";
+        query << "'" << type << "', ";
```
Comment (piotr-marcinkowski-red) [inline LISA/SqlDataStorage.cpp:274]: Please remember to change it to use prepared statements once other queries are reworked to use that solution

**Review by piotr-marcinkowski-red — APPROVED** [review]

File: LISA/Executor.cpp [hunk 3]
```diff
@@ -75,11 +51,27 @@ std::string generateHandle()
...
     ERROR_ALREADY_INSTALLED = 1003,
 };
 
+uint32_t Executor::Configure(const std::string& dbPath)
+{
+    auto result{ERROR_GENERAL};
+    try {
+        handleDirectories();
+        initializeDataBase(dbPath);
+        result = ERROR_NONE;
```
Comment (mariusz-sikora-red) [inline LISA/Executor.cpp:66]: Maybe set this at the beginning, so there will be no additional assignment. We return ERROR_GENERAL only when exception is fired.

**Review by mariusz-sikora-red — APPROVED** [review]

File: LISA/Executor.cpp (hunk 3 above)
Comment (szymon-polke-red) [reply LISA/Executor.cpp:66]: hm, yeah, I can change it in the next patch

**Pull request merged**